SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -1,8 +1,12 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
+<%(Callout type="warn" message="This [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
 # SpatialOS Unreal GDK toolbar
 
-The SpatialOS Unreal GDK toolbar is an easy way to run and alter [`spatial` command-line tool](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool) (SpatialOS documentation) commands from inside the Unreal Editor.
+The SpatialOS Unreal GDK toolbar provides several functions required for building and launching your client- and server-workers from inside the Unreal Editor.
```

Comment:
I'd change `the SpatialOS Unreal GDK toolbar` to `the toolbar for the SpatialOS GDK for Unreal` (because changing it to `the SpatialOS GDK for Unreal toolbar` is confusing because `Unreal` is right next to `toolbar`)

Reply:
Note: there are several other places where we still say "Unreal GDK" within this doc, but I haven't called each one out individually. I'm happy to update these if you like.